spec/FileLoggerAdapter.spec.js

```diff
@@ -59,7 +59,7 @@ describe('verbose logs', () => {
         level: 'verbose'
       });
     }).then((results) => {
-      expect(results[1].message.includes('"password": "********"')).toEqual(true);
```

Comment:
This is no longer testing the thing it should be testing, which is that passwords don't get output to the logs. The original version of this PR caused passwords to start getting output to logs, so I'd like to see these tests either not change, or be a lot more high-level/E2E and test the actual string that is logged rather than some intermediate object.